### PR TITLE
sync-diff-inspector: update sync_diff

### PR DESCRIPTION
**Pull request by Leavrth**
Signed-off-by: Leavrth <jianjun.liao@outlook.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
   - https://github.com/pingcap/tidb-tools/pull/675
   - https://github.com/pingcap/tidb-tools/pull/674
   - https://github.com/pingcap/tidb-tools/pull/664

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -102,8 +100,16 @@ check-struct-only = false
...
     password = ""
+
+    #（可选）使用 TLS 连接 TiDB
+    # security.ca-path = ".../ca.crt"
+    # security.cert-path = ".../cert.crt"
+    # security.key-path = ".../key.crt"
+
     #（可选）使用 TiDB 的 snapshot 功能，如果开启的话会使用历史数据进行对比
     # snapshot = "386902609362944000"
+    # 当 snapshot 设置为 "auto" 时，使用 cdc 在上下游的同步时间点，具体参考 https://github.com/pingcap/tidb-tools/issues/663
```
Comment (shichun-0415) [inline sync-diff-inspector/sync-diff-inspector-overview.md:111]:
```suggestion
    # 当 snapshot 设置为 "auto" 时，使用 TiCDC 在上下游的同步时间点，具体参考 <https://github.com/pingcap/tidb-tools/issues/663>
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**